src/components/Modal/Modal.tsx

```diff
@@ -87,7 +87,9 @@ export const Modal = ({
     focusableModalElements.current = modalRef.current.querySelectorAll(
       focusableElementSelectors
     );
-    focusableModalElements.current[0]?.focus();
+    // when the close button is rendered, we want to focus on the 2nd element
+    const index = close && focusableModalElements.current.length > 1 ? 1 : 0;
```

Comment:
Could you rewrite this as more explicit if/else? Using `&&` together with `? :` makes this quite hard to parse.